executor.go

```diff
@@ -104,7 +105,11 @@ func (r *Result) RowsAffected() (int64, error) {
 const prefix = "f"
 
 func executeStatement(ctx context.Context, config *config, query, transactionID string, args ...driver.NamedValue) (*Result, error) {
-	query = nameParameters(prefix, query)
+	if strings.Contains(config.database, "postgres") {
```

Comment:
Realistically there should be a better way of doing this... although I'm not sure what would make the most sense.